Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -81,6 +81,10 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
         ])
 
         webView.alpha = 0
+        
+        if #available(iOS 16.4, *) {
+            webView.isInspectable = true
```

Comment:
I'm not sure what's the implication of making a web view inspectable, but I decided to keep it off by default. The app enables it in the debug mode.

Reply:
I interpet "debug mode" to mean the scheme configuration for running the app. I have that enabled, but the web view does not display as inspectable for me in Safari. Am I missing something? 

| Xcode Scheme | Safari Inspector |
| - | -|
| <img width="937" alt="run-with-debug" src="https://github.com/wordpress-mobile/GutenbergKit/assets/438664/3702668e-1205-49ec-8f87-e08740b0542f"> | <img width="659" alt="no-inspectable-applications" src="https://github.com/wordpress-mobile/GutenbergKit/assets/438664/0e5358a5-331a-40e8-b771-f5b7dbcb034e"> |

Reply:
I moved this to the Demo if you don't mind – merging.